tests/test_download_topas.py

```diff
@@ -4,60 +4,68 @@
 import logging
 import subprocess
 from pathlib import Path
-from yaptide.admin.simulators import download_topas_from_s3
+from yaptide.admin.simulator_storage import download_topas_from_s3
 
 
 def check_if_environment_variables_set() -> bool:
     """Check if environment variables are set"""
     result = True
-    for var_name in ['S3_ENDPOINT', 'S3_ACCESS_KEY', 'S3_SECRET_KEY', 'S3_TOPAS_BUCKET', 'S3_TOPAS_KEY', 'S3_GEANT_BUCKET', 'S3_TOPAS_VERSION']:
+    for var_name in [
+            'S3_ENDPOINT', 'S3_ACCESS_KEY', 'S3_SECRET_KEY', 'S3_TOPAS_BUCKET', 'S3_TOPAS_KEY', 'S3_GEANT4_BUCKET',
+            'S3_TOPAS_VERSION'
+    ]:
         if var_name not in os.environ:
             logging.error(' variable %s not set', var_name)
             result = False
     return result
 
+
 @pytest.mark.skipif(sys.platform == "win32", reason="TOPAS does not work on Windows.")
 def test_if_topas_downloaded(tmpdir):
     """Check if TOPAS is downloaded and can be executed"""
     if check_if_environment_variables_set():
```

Comment:
Maybe it will be better to reverse this statement?
```suggestion
    if not check_if_environment_variables_set():
      pytest.fail("Required simulator variables were not set properly")
    else:
```

The rest of the code will be the happy case then, so we can drop `else`

Or maybe it will be better to use `pytest.fail` in fixture itself @grzanka?
Small article here: https://medium.com/opsops/pytest-fail-in-a-fixture-and-in-a-test-2122fab0564

Reply:
Here my idea was to skip the test if the environment variables are not set, instead of failing it.
This may be the case of local developer without S3 credentials.
The backend can work fine without S3 env variables, in such case demo version of SHIELD-HIT12A (and probably Fluka fake simulator as well) can be used.